qiskit_nature/operators/second_quantization/fermionic_op.py

```diff
@@ -144,70 +199,100 @@ class FermionicOp(SecondQuantizedOp):
 
     def __init__(
         self,
-        data: Union[str, Tuple[str, complex], List[Tuple[str, complex]]],
+        data: Union[
+            str,
+            Tuple[str, complex],
+            List[Tuple[str, complex]],
+            List[Tuple[str, float]],
+            List[Tuple[_FermionLabel, complex]],
+        ],
         register_length: Optional[int] = None,
-        sparse_label: bool = False,
+        display_format: str = "dense",
     ):
         """
         Args:
             data: Input data for FermionicOp. The allowed data is label str,
                   tuple (label, coeff), or list [(label, coeff)].
             register_length: positive integer that represents the length of registers.
-            sparse_label: the label is represented by sparse mode.
+            display_format: If sparse, the label of output is represented by sparse.
+                            if dense, the label of output is represented by dense. (default: dense)
 
         Raises:
             ValueError: given data is invalid value.
             TypeError: given data has invalid type.
         """
+        self.display_format: str
```

Comment:
I think we can make this `self._display_format` and then add `@property` and `@property.setter` methods instead of `set_display_format`

Reply:
This is a remnant from when it was first a class variable. You are right, that is better.